Create README.md for Hello Haptics

sriGanna opened the pull request:
This PR is to add a README file for the Hello Haptics Handler

Please ensure you've followed the checklist and provided all the required information *before* requesting a review.

## Required Information

- [ ] Reference the issue this is meant to fix or describe it if none exists.
- [ ] Full description of changes made.

## Pull Request Checklist

- [ ] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [ ] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [ ] The Docker image builds
- [ ] The container runs correctly when sent inputs with the changes
- [ ] No models or other large files are part of these commits
- [ ] A description of the tests already run as part of this PR is present
- [ ] CI is set up under `.github/workflows` or is not applicable

sriGanna committed:
Create README.md

sriGanna committed:
Update README.md

sriGanna committed:
Update README.md

JRegimbal reviewed (commented):
Reviewing since you didn't assign anyone

sriGanna committed:
Update handlers/hello-haptics-handler/README.md

Co-authored-by: Juliette Regimbal <juliette.regimbal@mailbox.org>

JRegimbal commented on handlers/hello-haptics-handler/README.md:
```diff
@@ -0,0 +1,6 @@
+
+## Hello Haptics Handler [DEPRECATED]
```
@sriGanna could you confirm that automatic builds should be disabled?

sriGanna replied:
sorry, yes it should be disabled

JRegimbal committed:
Never push hello haptics handler - deprecated

Also no longer run on schema updates since there's no real point to it.
Linting and test build kept just in case.
